man/resolved.conf.xml

```diff
@@ -152,7 +152,9 @@
         "downgrade" attacks, where an attacker might be able to
         trigger a downgrade to non-DNSSEC mode by synthesizing a DNS
         response that suggests DNSSEC was not supported. If set to
-        false, DNS lookups are not DNSSEC validated.</para>
+        false, DNS lookups are not DNSSEC validated and the resolver
+        becomes not security-aware. All forwarded queries have DNSSEC Okay
```

Comment:
"not security-aware" is strange wording. I think the second added part of this sentence is redundant; "DNS lookups are not DNSSEC validated" already says it all. 

"DNSSEC 0kay" is normally spelled "DNSSEC OK". "DNSSEC OK (DO)" would be even better.

Looks reasonable otherwise.

Reply:
Right, found exact wording of this bit on https://www.rfc-editor.org/rfc/rfc3225#section-3. Yes, it should be DNSSEC OK as in authoritative source.

Reply:
The original RFC 3225 uses mostly *DNSSEC-aware* term. Later RFC 4035 uses only *security-aware* term. Sometimes written without dash as *security aware*.

Reply:
> And no, `DNS lookups are not DNSSEC validated` does not say it all. It does not mention that any query received on stub listener will be forwarded _always_ with DO=0.

Well, but that's exactly what the *second* sentence says. But OK, we can have some redundancy, this doesn't really matter.